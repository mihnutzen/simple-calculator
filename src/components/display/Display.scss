@import 'src/styles/variables';

.Display {
  padding-right: 20px;
  height: $height;
  line-height: $height;
  background: #666;
  text-align: right;
  font-size: 8vh;
}