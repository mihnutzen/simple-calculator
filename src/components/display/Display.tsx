import React from 'react';

import './Display.scss';

const Display = ({ value = '0' }) => (
  <div className="Display">{value}</div>
);

export default Display;