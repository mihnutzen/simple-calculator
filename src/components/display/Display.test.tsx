import React from 'react';
import { render } from '@testing-library/react';

import Display from './Display';

describe('The Display component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Display />);
    const firstRender = asFragment()

    expect(firstRender).toMatchSnapshot();
  });

});
