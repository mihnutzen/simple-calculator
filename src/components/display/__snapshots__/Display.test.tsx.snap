// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Display component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="Display"
  >
    0
  </div>
</DocumentFragment>
`;
